.circleci/config.yml

```diff
@@ -120,9 +120,9 @@ workflows:
             - deploy-staging
 
       - node/run:
+          <<: *only_main
```

Comment:
This deploys the worker even before tests pass. I think it should instead be after everything (like `push-schema-changes`). In practice, when we make application-level changes coupled to worker-level changes, I'm not _sure_ which should deploy first. (We should probably avoid that.)

Reply:
Updated!